handlers/tokeninfo/jwt/tokeninfo.go

```diff
@@ -120,23 +128,35 @@ func newTokenInfo(t *jwt.Token, timeBase time.Time) (*TokenInfo, error) {
 	}, nil
 }
 
-func claimAsStrings(t *jwt.Token, claim string) ([]string, bool) {
+func NewTokenInfo(t *jwt.Token, timeBase time.Time) (*TokenInfo, error) {
```

Comment:
@szuecs I actuall use it in the "extended" internal tokeninfo that introduces a zalando specific JwtProcessor :(